connector-packager/connector_packager/helper.py

```diff
@@ -16,11 +21,11 @@ def check_jdk_environ_variable(exe_name):
             return True
 
     logger.error("Java Error: jdk_create_jar: no jdk set up in PATH environment variable, "
-                     "please download JAVA JDK and add it to PATH")
+                 "please download JAVA JDK and add it to PATH")
     return False
 
 
-def init_logging(log_path, verbose):
+def init_logging(log_path: str, verbose: Optional[bool]) -> logging.Logger:
```

Comment:
Why "**verbose: Optional[bool]**" vs "**verbose: bool = False**"?

Reply:
That's probably better, I _think_ that `None` is being passed by default sometimes? I'll dbl check in the IDE.

Reply:
fixed in f04d655